src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/MainCLIPrompt.java

```diff
@@ -16,23 +18,27 @@
 public final class MainCLIPrompt extends BaseCLIPrompt {
   private final AuthUseCase authUseCase;
   private final RegisterUseCase registerUseCase;
+  private final EmailSendUseCase emailSendUseCase;
 
-  public MainCLIPrompt(AuthUseCase authUseCase, RegisterUseCase registerUseCase) {
+  public MainCLIPrompt(AuthUseCase authUseCase, RegisterUseCase registerUseCase,
+      EmailSendUseCase emailSendUseCase) {
     this.authUseCase = authUseCase;
     this.registerUseCase = registerUseCase;
+    this.emailSendUseCase = emailSendUseCase;
   }
 
   private BaseCLIPrompt showRegisterOrEmailPrompt() {
     printDefault("What do you want to do?");
-    Map<String, BaseCLIPrompt> promptMap = new HashMap<>();
+    Map<String, BaseCLIPrompt> promptMap = new LinkedHashMap<>();
```

Comment:
I used a linkedhashmap here to always have reliable output sorting, improves muscle memory 🧠